dashboard/app/helpers/levels_helper.rb

```diff
@@ -471,6 +474,24 @@ def firebase_options
     fb_options
   end
 
+  def azure_speech_service_options
+    speech_service_options = {}
+
+    if @level.game.use_azure_speech_service?
+      uri = URI.parse("https://westus.api.cognitive.microsoft.com/sts/v1.0/issueToken")
```

Comment:
it isn't very likely, but we should use our service region key when we're constructing this URI. then, if it changes, our URI doesn't break. something like

```
uri = URI.parse("https://#{CDO.azure_speech_service_region}.api.cognitive.microsoft.com/sts/v1.0/issueToken")
```

should work, but double-check that i didn't make any mistakes there